source/HailPanel.cpp

```diff
@@ -55,25 +55,18 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship)
 		message = "(There is no response to your hail.)";
 	else if(!hasLanguage)
 		message = "(An alien voice says something in a language you do not recognize.)";
-	else if(gov->IsEnemy())
+	else if(gov->IsEnemy() && !ship->IsDisabled())
```

Comment:
Oddly, it looks like it might be a problem but I haven't replicated it. I'm not in a coding state of mind right now so I'll have to leave it until I am.